evaluator/logic.py

```diff
@@ -326,14 +326,12 @@ async def _insert_rule(self, rule: str, rule_cves: [str], conn: AsyncConnection,
             return row["id"]
 
     @time(EVAL_PART_TIME.labels(part="vmaas_request"))
-    async def _perform_vmaas_request(self, vmaas_req: str) -> (List[CveAdvisories], List[CveAdvisories], List[CveUnpatched]):
+    async def _perform_vmaas_request(self, vmaas_json: dict) -> (List[CveAdvisories], List[CveAdvisories], List[CveUnpatched]):
         """Perform VMAAS request for package based evaluation"""
         playbook_cves = []
         manually_fixable_cves = []
         unpatched_cves = []
```

Comment:
can `vmaas_json` be None? https://github.com/RedHatInsights/vulnerability-engine/blob/876b3837ab884e8a2a5071a89008c30a4fa4684e/evaluator/processor.py#L139
if so, it can crash on `if not vmaas_json.get("package_list", [])`
```suggestion
    async def _perform_vmaas_request(self, vmaas_json: dict | None) -> (List[CveAdvisories], List[CveAdvisories], List[CveUnpatched]):
        """Perform VMAAS request for package based evaluation"""
        playbook_cves = []
        manually_fixable_cves = []
        unpatched_cves = []
        
        if vmaas_json is None:
            return playbook_cves, manually_fixable_cves, unpatched_cves
```

Reply:
It can't happen, in `async def evaluate_vulnerabilities` (only function calling this) is this call under `if system_platform.vmaas_json:`